TicTacToe/src/tictactoe/TicTacToeAI.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package tictactoe;
+
+/**
+ *
+ * @author manxboy
+ */
+public class TicTacToeAI {
```

Comment:
The player is supposed to play as X (and go first).  In your game, the AI plays as X.